assets/sass/patterns/atoms/buttons.scss

```diff
@@ -97,3 +110,13 @@
 
   cursor: not-allowed;
 }
+
+.button--success {
+  background: $color-success;
+  border: 1px solid $color-success;
+
+  &:hover {
+    background-color: #2b7306;
```

Comment:
Does this colour belong in a variable?

Reply:
I have asked Chris if we are going to use this in the future elsewhere. If not then no point in creating a variable for it.

Reply:
Chris has replied saying we will not be using it elsewhere at this time so will leave it as it is.